README.md

```diff
@@ -135,9 +135,11 @@ Configuring the Prometheus exporter will create a server that listens on `promet
          `*` means that it will listen on all interfaces.
          Consider restricting to a given IP address -->
     <add key="prometheus_host" value="*" />
+	<add key="prometheus_https" value="false" />
```

Comment:
There are incorrect indentations on the other lines. Some are space and some are tabs - so i left it at tab instead of changing the whole section or continuing the discrepancy.